compensated-rails/compensated-rails.gemspec

```diff
@@ -0,0 +1,36 @@
+$:.push File.expand_path("lib", __dir__)
+
+# Maintain your gem's version:
+require "compensated/rails/version"
+
+# Describe your gem and declare its dependencies:
+Gem::Specification.new do |spec|
+  spec.name        = "compensated-rails"
+  spec.version     = Compensated::Rails::VERSION
+  spec.authors     = ["Tom"]
```

Comment:
There should be an author for `Zinc Collective LLC` and with a corresponding email of `hello@zinc.coop` so that the support burden doesn't fall on one person.